hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -1077,6 +1077,14 @@
     <value>86400000</value>
   </property>
 
+  <property>
+    <description>
+      RM delegation token remove-scan interval in ms
```

Comment:
Update comment

Reply:
Thanks for your suggestion, I will fix it.